tldr.py

```diff
@@ -296,11 +296,11 @@ def main():
                         action='store_true',
                         help='Downloads and caches all tldr pages from tldr site')
 
-    parser.add_argument('-o', '--os',
+    parser.add_argument('-p', '--platform',
                         nargs=1,
                         default=None,
                         type=str,
-                        choices=['linux', 'osx', 'sunos', 'windows'],
+                        choices=['linux', 'osx', 'sunos', 'windows', 'common'],
                         help="Override the operating system [linux, osx, sunos, windows]")
```

Comment:
does common need to be added to the help text as well?

Reply:
Yes, it should probably be added.